gui/src/ts/api/queries/benchmark/BenchmarkAnswerPageQuery.graphql

```diff
@@ -1,34 +1,40 @@
-query BenchmarkAnswerPageQuery($benchmarkId: String!, $questionId: String!, $datasetId: String!, $submissionId: String!) {
-    benchmarkById(id: $benchmarkId) {
-        name
-        datasetById(id: $datasetId) {
-            name
-            questionById(id: $questionId) {
-                choices {
-                    label
-                    text
-                }
-                text
-            }
-            submissionById(id: $submissionId) {
-                answerByQuestionId(id: $questionId) {
-                    choiceLabel
-                    explanation {
-                        choiceAnalyses {
-                            choiceLabel
-                            questionAnswerPaths {
-                                endNodeId
-                                startNodeId
-                                paths {
-                                    path
-                                    score
-                                }
-                            }
-                        }
-                    }
+query BenchmarkAnswerPageQuery(
+  $benchmarkId: String!
+  $questionId: String!
+  $datasetId: String!
+  $submissionId: String!
+) {
+  benchmarkById(id: $benchmarkId) {
+    name
+    datasetById(id: $datasetId) {
+      name
+      questionById(id: $questionId) {
+        choices {
+          label
+          text
+        }
+        text
+        correctChoiceLabel
+      }
+      submissionById(id: $submissionId) {
+        answerByQuestionId(id: $questionId) {
+          choiceLabel
+          explanation {
+            choiceAnalyses {
+              choiceLabel
+              questionAnswerPaths {
+                endNodeId
```

Comment:
You'll need node labels for these, I'll file an issue.

Reply:
Sounds good